firstTests.cy.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+
+describe('My first tests', () => {
+  it('should', () => {
+    cy.visit('https://react-redux.realworld.io')
+    
+    cy.contains('Sign in')
+      .click ()
+
+    cy.url()
+      .should('include', '/login')
+      cy.get(':nth-child(1) > .form-control')
+      .type('superemail8@email.ua')
+      cy.get(':nth-child(2) > .form-control')
+      .type('Superpassword8')
+      cy.get('.btn')
+      .click()
+      cy.get(':nth-child(4) > .nav-link')
+      .type('superusername8')
```

Comment:
why do you type smth here? it's better to check that there is user's username somewhere on the page to make sure that login has been successful.